esp/scm/ws_workunits.ecm

```diff
@@ -1690,14 +1690,16 @@ ESPrequest [nil_remove] WUCreateZAPInfoRequest
     string ProblemDescription;
     string WhatChanged;
     string WhereSlow;
-    [min_ver("1.53")] string Password;
+    [depr_ver("1.69")] string Password;
```

Comment:
depr_ver should be 1.70, meaning "deprecated, no longer exists as of 1.70".

Reply:
Changed